service/matching/db_task_queue_ownership.go

```diff
@@ -116,13 +134,21 @@ func (m *dbTaskQueueOwnershipImpl) getAckedTaskID() int64 {
 	m.Lock()
 	defer m.Unlock()
 
+	if m.status == dbTaskQueueOwnershipStatusLost {
+		return -1
```

Comment:
what is `-1`? 
try NOT to have this kind of special value

Reply:
Any negative value would do. It is to make sure no task is acked  when ownership is lost.